### ISSUE TITLE
Bug: the mouse pointer is shown above the html enabled tooltip window

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1785]
```
In the attached screenshot, you can see, the mouse pointer is placed above the tooltip
window. With this, it doesn't allow to hover the mouse on the adjacent Bars in the
column chart.

Actually, the tooltips should be displayed next to the mouse pointer. the default tooltips
works fine, but the problem is with the html enabled tooltips.

What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1.Create a Column Chart (i am using Microsoft VS 2013) with HTML enabled tooltips
2.On hover of the mouse pointer on the Bars/Columns, the tooltip gets displayed covering
the Bar and it's adjacent bars. Mouse pointer is placed above the tooltip. 
3. Doesn't give a chance to hover on the next bar.


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?Column Chart


Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)


What operating system and browser are you using?
Windows8, Google Chrome


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `biswajit.panda` on 2014-11-10 11:52:03

<hr>
- _Attachment: Tooltips.png<br>![Tooltips.png](https://storage.googleapis.com/google-code-attachments/google-visualization-api-issues/issue-1785/comment-0/Tooltips.png)_

**Comment by orwant** [comment]
```
This Bug prevents me to click(selectHandler) on the Bar. 

On click of the bar, i want to take the user to another page which i am not able to
do with html tooltip.

Please note : the default tooltip(non-html), works absolutely fine. the mouse cursor
is placed on the bar and the tooltip window is shown next/adjacent to the Bar. this
gives an opportunity to click the bar.


```

Original issue reported on code.google.com by `biswajit.panda` on 2014-11-14 06:29:46

**Comment by orwant** [comment]
```
This bug will be fixed with the next release. As a temporary workaround, you could either
set your tooltip to trigger on select. 

Alternatively, you could use the custom pivot option, which was introduced in the same
version (and is actually indirectly causing this bug), to explicitly position the tooltip
somewhere relative to the bar. Here's an example of it, making the tooltip appear to
the right of the bars: http://jsfiddle.net/mpyo5edm/
```

Original issue reported on code.google.com by `grabks@google.com` on 2014-11-14 14:46:19

**Comment by orwant** [comment]
```
When is the next release scheduled?

Based on your reply, I tried html tooltip with focusTarget: 'category', but i am not
getting expected result. Please take a look at the URL below:-

http://jsfiddle.net/pbiswajit/mpyo5edm/3/

Let me know if I am doing something differently/wrongly.
```

Original issue reported on code.google.com by `biswajit.panda` on 2014-11-17 07:35:31

**Comment by orwant** [comment]
```
try: 
            pivot: {
                x: 0,
                y: -120
            }
```

Original issue reported on code.google.com by `uilregor` on 2014-11-17 07:44:37

**Comment by orwant** [comment]
```
Oh, you're right. I'm sorry. The customPivot option won't work for focusTarget: 'category'.
We should cut the new release that fixes this issue at some point this week. We will
post to google-charts-announce when we do that: https://groups.google.com/forum/#!forum/google-charts-announce
```

Original issue reported on code.google.com by `grabks@google.com` on 2014-11-17 15:03:50

**Comment by orwant** [comment]
```
Dear Gra,

If I understand your response on 17th November correctly, you were supposed to put
a fix last week. I don't think it has been done.

Can you please let me know by when you have a plan to put a hotfix. The issue is causing
a lot of problem as I am unable to click on the bar/columns, as the mouse cursor always
remains on top of the the tooltip window. 
```

Original issue reported on code.google.com by `biswajit.panda` on 2014-11-25 04:17:30

**Comment by orwant** [comment]
```
We are always reluctant to specify dates for when we expect a release to be made, because
plans get changed. We are still finishing up the v40 release cycle (with some changes
internal to Google), and now expect we can start v41 early next week.  This is the
release (rather than a hotfix) that will have a fix for the tooltip issues.
```

Original issue reported on code.google.com by `dlaliberte@google.com` on 2014-11-25 14:12:15

**Comment by orwant** [comment]
```
From your response, it looks like I have to wait for a month or even more to get my
issue addressed. Based on the brand value, I had relied on this API. 

If an existing feature is broken due to recent release, you must put a hotfix so that
those who are already using your APIs don't get effected. 

Please suggest....
```

Original issue reported on code.google.com by `biswajit.panda` on 2014-11-26 05:57:08

**Comment by orwant** [comment]
```
Any update on this Please?

When are you going for release... It's really troubling us.
```

Original issue reported on code.google.com by `biswajit.panda` on 2015-02-06 07:38:06

**Comment by orwant** [comment]
```
This is now released in 1.1. If you are continuing to see this issue, please post back
with an example reproducing it, and we will investigate.

Thank you for your patience,
- Sergey
```

Original issue reported on code.google.com by `grabks@google.com` on 2015-02-06 15:01:51

**Comment by orwant** [comment]
```
I did a quick sanity test to evaluate the fix and found to be working fine. 

Thank you very much.
```

Original issue reported on code.google.com by `biswajit.panda` on 2015-03-04 06:23:20
